btools/building/door/door_types.py

```diff
@@ -128,16 +128,7 @@ def create_door_fill(bm, face, prop):
 def fill_door_face(bm, face, prop):
```

Comment:
This function is useless now. remove it and call `fill_face` where it is called.